### PR TITLE
Add a "get_filename" method

### PR DESCRIPTION
**Pull request by lbalogh**
Hi,

As discussed in issue https://github.com/willprice/python-omxplayer-wrapper/issues/26, add a "get_filename" method to get the current playing filename

### CODE REVIEW

File: omxplayer/player.py [hunk 1]
```diff
@@ -61,6 +61,7 @@ def __init__(self, filename,
 
         self.tries = 0
         self._is_playing = True
+	self._filename = filename
```
Comment (willprice) [inline omxplayer/player.py]: Whitespace looks a bit off, make sure you're using tabs instead of spaces

File: omxplayer/player.py [hunk 2]
```diff
@@ -428,6 +429,14 @@ def quit(self):
...
         except OSError:
             logger.error('Could not find the process to kill')
+ 
+    @_check_player_is_active
+    def get_filename(self):
+        """
+        Returns:
+            str: filename currently playing
+        """
+	return str(self._filename)
```
Comment (willprice) [inline omxplayer/player.py]: Since we're passing in the filename there is no need to call `str` on it, filenames are inherently strings.

**Comment by willprice** [comment]
Thanks for your contribution :)

**Comment by lbalogh** [comment]
Oups, looks like I have some problem with my editor... :/

**Pull request merged**